### PR TITLE
add 1.1.0 release

### PR DESCRIPTION
**Pull request by jahubba**

### CODE REVIEW

File: latest/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ MAINTAINER Eric Bottard
 
 # 'snapshot' or 'release'
 ENV XD_BUILD snapshot
-ENV XD_VERSION 1.0.2.BUILD-SNAPSHOT
+ENV XD_VERSION 1.1.0.RELEASE
```
Comment (ericbottard) [inline latest/Dockerfile]: This should be 1.2.0.BUILD-SNAPSHOT

**Comment by ericbottard** [comment]
Thank you for this contribution. Before we can accept it though, could you please sign the ICLA at https://support.springsource.com/spring_committer_signup (selecting Spring XD as the project) and leave a comment here once this is done.

**Comment by jahubba** [comment]
Thanks for reviewing, I have signed the ICLA and updated the image-base for latest to the current snapshot.

**Comment by ericbottard** [comment]
Applied all changes and created new tags on docker-hub. Thanks again for this contribution!

**Pull request merged**